src/components/Modals/ManageCollectionRoleModal/EmptyRole/EmptyRole.tsx

```diff
@@ -22,24 +23,29 @@ export default class EmptyRole extends React.PureComponent<Props, State> {
     onCancel()
   }
 
-  handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+  handleChange = (_: React.ChangeEvent<HTMLInputElement>, { value }: InputOnChangeData) => {
```

Comment:
Isn't changing the `handleChange` method here breaking the old Field `onChange` handler?

Reply:
it shouldn't , its just changing where it takes the value from but the function signature is the samae one for previous Field component